### PR TITLE
Release/small components 2

### PR DESCRIPTION
**Pull request by NirK8**
card component styling, beginning of a nav bar

### CODE REVIEW

**Comment by NirK8** [comment]
Screen Shot 2021-06-29 at 12 19 57

File: header.php [hunk 1]
```diff
@@ -6,9 +6,21 @@
 </head>
 <body>
   <header>
-    <!-- <div class="logo-container">
       <a href="/dispatcher">
-        <img class="logo" src="< ?php echo get_template_directory_uri() . "/img/logo.jpeg" ?>"/>
+        <img class="logo" src="<?php echo get_template_directory_uri() . "/images/logo.png" ?>"/>
```
Comment (Aslanchik-m) [inline header.php:10]: Logos and Icons should be exported as SVG and not png or jpeg.
Reference -
[5 reasons you should be using SVG over PNG](https://www.growfox.co.uk/blog/5-reasons-you-should-be-using-svgs-over-pngs)

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const onSearchButtonClicked = (e) => {
+  console.log('value: ', searchInput.value);
```
Comment (Aslanchik-m) [inline js/scripts.js:2]: Before pushing code to a pull request, remove all unnecessary code and console.logs.

File: sass/style.scss [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    Description: Theme for the dispatcher
+    Version: 1.0
+    License: GNU Public License v2 or later
+    License URI: http://www.gnu.org/licenses/glp-2.0.html
+    Tags: pokemon, mobile first, responsive, dispatcher
+    Text Domain: dispatcher
+*/
+
+@import "compass";
+@import "compass/reset";
```
Comment (Aslanchik-m) [inline sass/style.scss:15]: We talked about breaking down the scss file into smaller files. Just a reminder for the next pull request.

File: sass/style.scss [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+  --light-pink: #F3F3FF;
+  --grey: #D9DBE9;
+  --light-opaque-grey: rgba(217,219,233, 0.5);
+  --opaque-grey: rgba(217,219,233, 0.8);
+  --blue: #0058B9;
+  --light-purple: #5A5A89;
+  --light-opaque-purple: rgba(90, 90, 137, 0.5);
+  --dark-purple: #262146;
+  --black: #14142B;
+  --white: #FFFFFF;
```
Comment (Aslanchik-m) [inline sass/style.scss:29]: Create a global colors.scss and store all your colors as variables there.
Try giving more specific names, like primary-dark, primary-grey, secondary-grey etc..

File: sass/style.scss [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+  padding: 0 20px;
+  background-color: var(--dark-purple);
+  .logo {
+    height: 50px;
+  }
+  .search-container {
+    display: flex;
+    align-items: center;
+    border-radius: 10px;
+    background: #FFFFFF;
```
Comment (Aslanchik-m) [inline sass/style.scss:45]: If you have a variable for white, use it here.
Avoid hardcoding colors whenever possible, you may never know how designs will change.

File: sass/style.scss [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+    width: 423px;
+    height: 50px;
+    > span {
```
Comment (Aslanchik-m) [inline sass/style.scss:48]: I think > is not needed here, you can probably just write span.

File: sass/style.scss [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+              border-radius: 10px;
+              margin: 0px 10px;
+              background-color: var(--light-pink);
+              font-family: 'Poppins', sans-serif;
+              color: var(--light-purple);
+              cursor: pointer;
+            }
+            .tag:hover {
+              background-color: var(--opaque-grey);
+            }
```
Comment (Aslanchik-m) [inline sass/style.scss:171]: Instead, you can write it in the sass way:
`&:hover {
    background-color: var(--opaque-grey);
}`

Comment (Aslanchik-m) [reply sass/style.scss:171]: Great reference for the `&`
[The Sass Ampersand](https://css-tricks.com/the-sass-ampersand/)

File: sass/style.scss [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+            line-height: 21px;
+            color: var(--black);
+            margin: 10px 0;
+          }
+          .author-and-source {
+            color: var(--light-opaque-purple);
+            margin: 10px 0;
+            line-height: 22px;
+            max-height: 22px;
+            letter-spacing: 0.25px;
```
Comment (Aslanchik-m) [inline sass/style.scss:189]: Letter spacing should also be in rem or em.

File: sass/utils/variables.scss [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+$light-pink: #F3F3FF;
+
+$primary-light: #5A5A89;
+$primary-opaque: rgba(90, 90, 137, 0.5);
+$primary-dark: #262146;
+
+
+$black: #14142B;
+$white: #FFFFFF;
```
Comment (Aslanchik-m) [inline sass/utils/variables.scss:17]: Good job!

**Review by Aslanchik-m — APPROVED** [review]

**Comment by Aslanchik-m** [comment]
Well done Nir, keep up the good work!

**Pull request merged**